src/box/lua/config/utils/schema.lua

```diff
@@ -1599,6 +1606,83 @@ end
 
 -- }}} <schema object>:pairs()
 
+-- {{{ <schema object>:jsonschema()
+
+-- This function sets common fields such as `description`, `default`, and `enum`
+-- for a given schema object. These fields are typical in JSON schemas.
+local function set_common_jsonschema_fields(res, schema)
+    res.description = schema.description
+    res.default = schema.default
+    res.enum = schema.allowed_values
+    return setmetatable(res, {
+        __serialize = 'map',
+    })
+end
+
+local function jsonschema_impl(schema, ctx)
+    if is_scalar(schema) then
+        -- If the schema is a scalar type (string, number, etc.)
+        -- copy the corresponding JSON schema.
+        local scalar_copy = table.copy(scalars[schema.type].jsonschema)
+        return set_common_jsonschema_fields(scalar_copy, schema)
+    elseif schema.type == 'record' then
+        local properties = {}
+        for field_name, field_def in pairs(schema.fields) do
+            walkthrough_enter(ctx, field_name)
+            if type(field_name) ~= 'string' then
+                walkthrough_error(ctx, 'JSON does not support non-string keys.')
```

Comment:
Nit: All the other errors in this module have no period at the end.

(The same about the next error in this function.)

Reply:
Done.